src/rules/selector-no-vendor-prefix/index.js

```diff
@@ -22,11 +22,11 @@ export default function (actual) {
 
       // Check each pseudo-selector
       styleSearch({ source: selector, target: ":" }, match => {
-        const pseudoSelector = /:{1,2}[a-z-]+\b/.exec(selector.slice(match.startIndex))[0]
-
-        if (isAutoprefixable.selector(pseudoSelector)) {
+        const pseudoSelector = /:{1,2}[a-z-]+\b/.exec(selector.slice(match.startIndex))
+        if (pseudoSelector === null) { return }
+        if (isAutoprefixable.selector(pseudoSelector[0])) {
```

Comment:
This is an improvement, yes. And if it fixes the problem and all tests still pass, go with it.